src/components/change-default-method/change-default-method-controllers.ts

```diff
@@ -66,11 +104,16 @@ export async function changeDefaultMethodAppGet(
   );
 }
 
-const setLocalOplSettings = (res: Response) => {
+const setChangeDefaultMethodSmsOplSettings = (res: Response) => {
   setOplSettings(
-    {
-      taxonomyLevel2: "change phone number",
-    },
+    supportMfaManagement()
+      ? {
+          ...MFA_COMMON_OPL_SETTINGS,
+          contentId: "e847d040-e59e-4a88-8f9c-1d00a840d0bd",
+        }
+      : {
+          taxonomyLevel2: "change phone number",
```

Comment:
"change phone number" - could we have `const` as it reused in src/components/update-confirmation/update-confirmation-controller.ts & src/components/enter-password/enter-password-controller.ts?